### PR TITLE
Changed Finding URL Section for Consolidated Findings

### PR DESCRIPTION
**Pull request by maira-samtek**

### CODE REVIEW

File: src/libs/security-hub-lib.ts [hunk 1]
```diff
@@ -5,14 +5,31 @@ import {
...
   AwsSecurityFinding,
-  Resource,
   AwsSecurityFindingFilters
 } from '@aws-sdk/client-securityhub'
 import {SecurityHubJiraSyncConfig} from '../macfc-security-hub-sync'
 import {extractErrorMessage} from '../index'
 
+export interface Resource {
+  Type: string | undefined
+  /**
```
Comment (yonassrobi) [inline src/libs/security-hub-lib.ts]: Is this comment needed?

File: src/libs/security-hub-lib.ts [hunk 2]
```diff
@@ -5,14 +5,31 @@ import {
...
+   * @public
+   * <p>The canonical identifier for the given resource type.</p>
+   */
+  Id: string | undefined
+  /**
+   * @public
+   * <p>The canonical Amazon Web Services partition name that the Region is assigned to.</p>
+   */
```
Comment (yonassrobi) [inline src/libs/security-hub-lib.ts]: Same as above? Since the other variables do not have comments, it might be cleaner to remove them.

**Review by yonassrobi — APPROVED** [review]
LGTM. Add a minor comment to remove comments that might not be needed.

File: src/libs/security-hub-lib.ts (hunk 2 above)
Comment (maira-samtek) [reply src/libs/security-hub-lib.ts]: removed

**Comment by maira-samtek** [comment]
> LGTM. Add a minor comment to remove comments that might not be needed.

@yonassrobi I removed comments. Also, following are links of example issues with screenshots attached
[enterprise jira ticket](https://jiraent.cms.gov/browse/CMCSMACD-3137)
[atlassian jira ticket](https://mairasamtek0.atlassian.net/browse/TT-5823)

image
image

**Pull request merged**